### DISCUSSION TITLE
cross product

### DISCUSSION THREAD

**Discussion by sed-i** [rust-ndarray/ndarray#1140] [Q&A] [answered]
(coming from numpy and without much knowledge in rust)
I was wondering if there was a particular reason why cross product is not implemented, and if it's possible to implement it only for `Array1<A> = Array<A, Ix1>`.

**Comment by adamreichold** [comment, marked as answer]
As you note, this would be limited to three-dimensional vectors as the underlying isomorphism of 2-forms and vectors only works in this case. I think this why appears to be more of the territory of libraries with a stronger focus on computer graphics like [`nalgbra`](https://nalgebra.org/) which does support this and has support for converting to/from `ndarray` types.
